portfolio/src/main/data_proccessing/requirements.

```diff
@@ -0,0 +1,25 @@
+cachetools==4.1.1
```

Comment:
Shouldn't this be called requirements.txt (you're missing the file extension).

Reply:
Okay sure, I'll fix that.